packages/patternfly-4/react-core/src/components/Wizard/Wizard.tsx

```diff
@@ -56,11 +58,11 @@ export interface WizardProps extends React.HTMLProps<HTMLDivElement> {
   /** Custom height of the wizard */
   height?: number | string;
   /** The wizard title */
-  title: string;
+  title?: string;
```

Comment:
title is now optional, because the header is not rendered in in-page wizard

Reply:
The default wizard requires a title currently, and per conversations with @mcarrano here (https://github.com/patternfly/patternfly-next/issues/2191) I believe we want to keep it that way. Making it optional creates these 2 scenarios:

Title/description is undefined renders an empty header, and the header is required at a minimum to position the close button.

<img width="1130" alt="Screen Shot 2019-09-05 at 9 18 11 AM" src="https://user-images.githubusercontent.com/35148959/64350539-7084d480-cfbe-11e9-98ab-1fd986b62178.png">

Title is undefined but a description exists, which seems to work OK, but we haven't really accounted for that in core and I'm not sure we want to allow that per the design:

<img width="1134" alt="Screen Shot 2019-09-05 at 9 21 06 AM" src="https://user-images.githubusercontent.com/35148959/64350599-8b574900-cfbe-11e9-993f-2348195f7e42.png">

Can we make this optional if `isInPage` is `true`? And technically it isn't optional - the entire header shouldn't render if using an in-page wizard, so `title` and `description` shouldn't be allowed.

Reply:
We can change the title to optional here since it technically is not always required.  However, can you generate a warning if the title is not provided AND isModal is true.